sources/documents/2_RTB/Verbali/Interni/VerbaleInterno_2024-07-18/todo.tex

```diff
@@ -0,0 +1,39 @@
+\section{Todo / In Progress}
+
+\bgroup
+\begin{center}
+  \begin{longtable}{|P{1.9cm}|P{5.2cm}|P{3cm}|>{\arraybackslash}P{2.4cm}|}
+    \hline
+    \textbf{Ticket} & \textbf{Incarico} & \textbf{Incaricato/a} & \textbf{Scadenza}\\
+    \hline
+    \endfirsthead
+
+    \hline
+		\textbf{Ticket} & \textbf{Incarico} & \textbf{Incaricato/a} & \textbf{Scadenza} \\
+		\hline
+		\endhead
+
+    \multicolumn{4}{|r|}{{Continua nella prossima pagina}} \\ 
+		\hline
+		\endfoot
+
+    \hline
+		\endlastfoot
+    
+    \hline ARGO-221 & Espansione dei processi di supporto nelle \NdP & \raul & 2024-07-19 \\
+    \hline ARGO-222 & Stesura consuntivo \glossario{sprint} 8 & \mattia & 2024-07-19 \\
+    \hline ARGO-223 & Revisione sezione rischi nelle \PdP & \riccardo & 2024-07-19 \\
+    \hline ARGO-224 & Verbale interno 2024-07-18 & \raul & 2024-07-20 \\
+    \hline ARGO-226 & Aggiornamento sezione verifica nelle \NdP & \riccardo & 2024-07-19 \\
+    \hline ARGO-227 & Aggiornamento sezione Validazione nelle \NdP & \riccardo & 2024-07-19 \\
+    \hline ARGO-230 & Aggiornamento sezione strumenti nelle \NdP & \tommaso & 2024-07-19 \\ 
+    \hline ARGO-231 & Stesura sezione \glossario{CSS} nelle \NdP & \sebastiano & 2024-07-19 \\
+    \hline ARGO-232 & Stesura iniziale delle specifiche tecniche & \riccardo & 2024-07-20 \\
+    \hline ARGO-233 & Individuazione pattern architetturali per il \glossario{back-end} & - & - \\
```

Comment:
Queste vanno rimosse, sono nel backlog, ma non ancora uno sprint di riferimento.

Reply:
Da ARGO-230 in giù.